zerver/lib/request.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, whence: str=None, *, converter: Callable[[Any], Any]=None,
 # expected to call json_error or json_success, as it uses json_error
 # internally when it encounters an error
 def has_request_variables(view_func):
-    # type: (Callable[[HttpRequest, Any, Any], HttpResponse]) -> Callable[[HttpRequest, *Any, **Any], HttpResponse]
+    '''
+    type: (Callable[[HttpRequest, Any, Any], HttpResponse]) ->
+    Callable[[HttpRequest, *Any, **Any], HttpResponse]
+    '''
```

Comment:
This is the wrong fix -- we should instead be fixing this type annotation, not changing it to a docstring.